Adds CSS blogpost (#1)
MinThaMie/anne-greeth: content/css-blog.md

---
title: When CSS seems scary
image: /images/writing.jpg
imageMeta:
  attribution:
  attributionLink:
featured: true
authors:
  - anne-greeth-schot-vanherwijnen
date: Wed Oct 6 2021
tags:
  - css
  - tech
  - beginner
---

> In this blogpost I want to show you how I tackle CSS. There are so many CSS rules, so many properties and options that it might feel overwhelming. I would recommend you to start small. 

Reviewer: ```suggestion
In this blogpost I want to show you how I tackle CSS. There are so many CSS rules, so many properties and options that it might feel overwhelming. I would recommend that you start small. 
```

If we take the example of a HTML button element. You might like it the way it is or you might prefer to give it a bit more charm.  

# Tip 1: Open the browser

> Often when I want to (re)design something I open up the dev-tools in the browser by right clicking “inspect element” and I’ll just write my CSS right there. The browser will give you auto-completion and a ton of options and you can see your changes directly.

Reviewer: ```suggestion
Often when I want to (re)design something I open up the dev-tools in the browser by right clicking and choosing “inspect element” and I’ll just write my CSS right there. The browser will give you auto-completion and a ton of options and you can see your changes directly.
```

Reviewer: but also... I think depending on who you're aiming this at you might need to explain "and I'll just write my CSS right there" 🤔 you could even put a gif in here to show you right-clicking and editing a bit of CSS in the right place, what do you think?

# Tip 2: Step by step

> Don’t think you have to know right away where your design will go. You can just start bit by bit changing what is already there. I usually start with some spacing, the height/width and color.

Reviewer: this also feels like it might be good with an example

# Tip 3: Steal like an artist

> Luckily a lot of people with CSS experience have built awesome codepens and libraries. You can steal from those by just copy-pasting or steal from them by understanding and adapting what they made. Often when you try to describe what you are looking for you will either find some code that does exactly that, learn better the different building block of what you want to built (and can probably start building yourself) or know what your next google search should be. 

Reviewer: ```suggestion
Luckily a lot of people with CSS experience have built awesome codepens and libraries. You can steal from those by just copy-pasting or steal from them by understanding and adapting what they made. Often when you try to describe what you are looking for you will either find some code that does exactly that, learn more about the different building blocks of what you want to built (and can probably start building yourself), or know what your next google search should be. 
```

# Tip 4: Kill the perfectionist

> The difficult thing with design is that we know our taste and we know what others are capable of. This might increase the bar to start writing your CSS, because somebody else might know a better or more beautiful solution. However, they also had to learn how to get there. If your CSS does what it needs to and looks the way you like (at this point in time), you’ve done it! You wrote CSS!

Reviewer: What do you mean by `might increase the bar to start writing your CSS`? How about "put up a hurdle for writing your CSS"? is that what you mean?

# Bonus tip: Use resources to learn
The [MDN docs on CSS](https://developer.mozilla.org/en-US/docs/Web/CSS) have explanations of all the properties and what parameters you can and cannot pass. Also CSS-tricks is your friend, where MDN has the definition CSS-tricks has often more of the implementation.  You can use these resources to learn, or of course to re-learn, because there is no shame in finding out how to flex box more than once

> I hope these tips will allow you to start feel more comfortable with CSS, and remember that when CSS seems scary you can do it anyway!

Reviewer: ```suggestion
I hope these tips will allow you to start feeling more comfortable with CSS, and remember that when CSS seems scary you can do it anyway!
```
